src/material/schematics/ng-generate/mdc-migration/rules/components/slider/slider-template.spec.ts

```diff
@@ -97,19 +97,74 @@ describe('slider template migrator', () => {
     );
     await runMigrationTest(
       `
-      <mat-slider vertical invert></mat-slider>`,
+      <mat-slider tickInterval></mat-slider>`,
```

Comment:
I think that `tickInterval` usually has a value. Should the test reflect that? Same for the other tests added in the PR.